cephadm-preflight.yml

```diff
@@ -213,6 +213,15 @@
                 filename: ceph
                 update_cache: no
 
+            - name: configure Ceph custom repositories
+              when: ceph_origin == 'custom'
+              apt_repository:
+                repo: "deb {{ item.baseurl }}/ {{ ansible_facts['distribution_release'] }} {{ item.components }}"
```

Comment:
`item.components` will be a mandatory parameter for debian-based OS only, right?

Reply:
Yes ,`item.components `debian-based OS only.
When this is merged I will update `item.components ` doc.

Reply:
> When this is merged I will update doc for debian-based variable ( `item.components `) .

why don't you update the documentation in this PR?

Reply:
I found out that commits to doc are done separately for this project. 
I added this variable in doc. 
What else should I add or change?

Reply:
> I found out that commits to doc are done separately for this project.

That's true I'm personally used to sending separate commits for the documentation but this project has no such requirements.